<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>140</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1400</width>
     <height>1000</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1400</width>
     <height>1000</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget {
	background-color: rgb(#12372A);
}</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1400</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1400</width>
      <height>1000</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1400</width>
      <height>1000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#stackedWidget {

	background-color: &quot;#59736a&quot;;

}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="start_page">
     <property name="styleSheet">
      <string notr="true">background-color: &quot;#0e2c22&quot;;</string>
     </property>
     <widget class="QLabel" name="logo_label">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>150</y>
        <width>581</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);	
font: 75pt ;</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Chess Match</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="start_page_groupBox">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>350</y>
        <width>521</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: &quot;#ADBC9F&quot;;
border-radius: 10px;

</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>402</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { 
	border-radius: 10px;	
	background-color: &quot;#12372A&quot;;
	color: rgb(255, 255, 255);	
	font: 22pt ;
}

QPushButton:hover { 
	
	color: &quot;#FF9C42&quot;;
	background-color: &quot;#415f55&quot;;

}</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="registerButton">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { 
	border-radius: 10px;	
	background-color: &quot;#12372A&quot;;
	color: rgb(255, 255, 255);	
	font: 22pt ;
}

QPushButton:hover { 
	
	color: &quot;#FF9C42&quot;;
	background-color: &quot;#415f55&quot;;

}</string>
          </property>
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exitButton">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { 
	border-radius: 10px;	
	background-color: &quot;#12372A&quot;;
	color: rgb(255, 255, 255);	
	font: 22pt ;
}

QPushButton:hover { 
	
	color: &quot;#FF9C42&quot;;
	background-color: &quot;#415f55&quot;;

}</string>
          </property>
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="authorization_page">
     <property name="styleSheet">
      <string notr="true">background-color: &quot;#0e2c22&quot;;</string>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>290</y>
        <width>941</width>
        <height>631</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: &quot;#ADBC9F&quot;;
border-radius: 10px;
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>470</y>
         <width>202</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="registerRegButton">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { 
	border-radius: 10px;	
	background-color: &quot;#12372A&quot;;
	color: rgb(255, 255, 255);	
	font: 18pt ;
}

QPushButton:hover { 
	
	color: &quot;#FF9C42&quot;;
	background-color: &quot;#415f55&quot;;

}</string>
          </property>
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="regReturnButton">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { 
	border-radius: 10px;	
	background-color: &quot;#12372A&quot;;
	color: rgb(255, 255, 255);	
	font: 18pt ;
}

QPushButton:hover { 
	
	color: &quot;#FF9C42&quot;;
	background-color: &quot;#415f55&quot;;

}</string>
          </property>
          <property name="text">
           <string>Return</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>631</width>
         <height>321</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="nickname_reg_label">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);	
font: 25pt ;</string>
            </property>
            <property name="text">
             <string>Nickname</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nicknameRegLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);	
background-color: &quot;#FBFADA&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="password_reg_label">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);	
font: 25pt ;</string>
            </property>
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passwordRegLineEdit">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);	
background-color: &quot;#FBFADA&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="password_confirm_reg_label">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);	
font: 25pt ;</string>
            </property>
            <property name="text">
             <string>Password confirmation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passwordConfirmRegLineEdit">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);	
background-color: &quot;#FBFADA&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="authorization_label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>220</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);	
font: 35pt ;</string>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="profile_page"/>
    <widget class="QWidget" name="rating_page"/>
    <widget class="QWidget" name="game_page">
     <widget class="QGraphicsView" name="pawnTransformView">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>61</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="boardGraphicsView">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>651</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="opponentTakenView">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>100</y>
        <width>261</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="playerTakenView">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>250</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="game_info_label">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>40</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>game_info</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NewGameButton">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>430</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="board_style_label">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>490</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="player_color_label">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>520</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="BoardStyleComboBox">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>480</y>
        <width>86</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="PlayerColorComboBox">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>520</y>
        <width>86</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
